application/ai/tension_analyzer_llm_contract.py

```
@@ -0,0 +1,100 @@
+"""张力分析器：LLM JSON 契约、解析校验与 OpenAI-style tool 定义。
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)